model/src/main/proto/javaclasses/mealorder/c/events.proto

```diff
@@ -107,3 +109,86 @@ message OrderProcessed {
     reserved 3 to 10;
 }
 
+// An event reflecting the vendor addition.
+//
+message VendorAdded {
+
+    // An identifier of the added Vendor.
+    VendorId vendor_id = 1;
+
+    // An identifier of the User, who added Vendor.
+    UserId user_id = 2;
+
+    // Time of addition.
+    google.protobuf.Timestamp when_added = 3;
+
+    // Vendor description.
+    VendorDescription vendor_description = 4;
+
+    // Reserved for future extension.
+    reserved 5 to 10;
```

Comment:
Why do we need it? This makes sense when you reserve a range _in between_ of already defined range. But why here?

Reply:
@Oleg-Melnik, please address the above comment.

Reply:
@alexander-yevsyukov ,we sorted it out at a meeting yesterday. Redundant reservations will be deleted.